src/main/api/ledger.ts

```diff
@@ -110,14 +111,55 @@ const signBtcTxInLedger = async (
   }
 }
 
+const signBNBTxInLedger = async (ledgerTxInfo: LedgerBNBTxInfo): Promise<E.Either<LedgerErrorId, string>> => {
+  const {
+    client,
+    sender,
+    asset,
+    amount,
+    recipient,
+    memo,
+    handleLedgerPresign,
+    handleLedgerVerifySuccess,
+    handleLedgerVerifyFailed
+  } = ledgerTxInfo
+  // await client.initChain()
+  const transport = await TransportNodeHid.open('')
+  const ledgerApp = new LedgerAppBNB(transport)
+  const derive_path = getBNBDerivePath(0)
+  client.useLedgerSigningDelegate(
```

Comment:
`useLedgerSigningDelegate` is a function we do need to implement into `xchain-binance`, right?